src/UI/MainMenuOptions.h

```diff
@@ -68,6 +68,5 @@ class MainMenuOptions : public Window
 
     bool optionsChanged = false;
     bool videoOptionsChanged = false;
-    bool inInit = true;
```

Comment:
Kind of a minor point for a PR, though it looks like this was removed in a commit before all uses of it were removed. That means the commit with a partial removal can not build.